### ISSUE TITLE
ValidationError({0: {'percentage-change': ['Unknown field.']}})

### ISSUE THREAD

**Issue by utkarshski** [chartmogul/chartmogul-python#66]
While retrieving the MRR,

```python
chartmogul.Metrics.mrr(
config,
start_date="2021-12-01",
end_date="2022-06-21",
interval="month")
```
I'm getting this error message:
```python
<Promise at 0x1622921c0 rejected with ValidationError({0: {'percentage-change': ['Unknown field.']}, 1: {'percentage-change': ['Unknown field.']}, 2: {'percentage-change': ['Unknown field.']}, 3: {'percentage-change': ['Unknown field.']}, 4: {'percentage-change': ['Unknown field.']}, 5: {'percentage-change': ['Unknown field.']}, 6: {'percentage-change': ['Unknown field.']}})>
```

**Comment by SrikalaB** [comment]
@utkarshski Can you add more details about the version you are using here?
We had a bug in 3.0.1 and the fix is in v3.0.2. Can you upgrade to the latest version and try - https://pypi.org/project/chartmogul/3.0.2/

**Comment by utkarshski** [comment]
@SrikalaB,
Ah, It's working fine,
Though I upgraded it earlier, it wasn't working, had to restart the kernel.
